src/msg.h

```diff
@@ -93,6 +93,7 @@ typedef struct {
 typedef struct {
 	uint8_t  type;       /**< Must be \ref BL_MSG_START */
 	uint16_t frequency;  /**< Sampling rate in Hz. */
+	uint16_t led_mask;   /**< Mask of LEDs to use. */
 	uint16_t src_mask;   /**< Mask of sources to enable. */
 } bl_msg_start_t;
```

Comment:
I think we need to have an acquisition mode parameter.

Something like in `acq.h`:

```c
/** Acquisition mode. */
enum bl_acq_mode {
	BL_ACQ_MODE_CONTINUOUS,
	BL_ACQ_MODE_FLASH,
};
```

And:

```suggestion
typedef struct {
	uint8_t  type;       /**< Must be \ref BL_MSG_START */
	uint8_t  mode;       /**< See \ref enum bl_acq_mode */
	uint16_t frequency;  /**< Sampling rate in Hz. */
	uint16_t led_mask;   /**< Mask of LEDs to use. */
	uint16_t src_mask;   /**< Mask of sources to enable. */
} bl_msg_start_t;
```

This will enable us to have a continuous-mode acquisition, with for example red and green LEDs on.

Reply:
Just a little bit unsure, will we ever have the usage case of having multiple LEDs on at same time, and run it in continuous mode? Won't the LEDs interfere different PDs? No matter how weak it will be all PDs are still picking up lights from all LEDs, right?

Reply:
Yeah, but sometimes I've done it to see a stronger pulse!

Reply:
Ok, can make the change